### ISSUE TITLE
Erldash can't connect to node if EMDP isn't running

### ISSUE THREAD

**Issue by Zabrane** [sile/erldash#11]
Hi @sile

We ran our Erlang releases without `empd` for security reasons.
In this case, `erldash` (0.2.0) can't connect to collect the node metrics.

Is it possible to enhance `erldash` to let connect to empdless node using `remsh`?

This is the error I'm getting when trying to connect to an epmdless node
```
> erldash run --cookie 8nL2aZ rune@127.0.0.1
Error: unexpected end of file
```

**Comment by sile** [comment]
I think that it would be worth considering to support epmdless nodes. 
However, I'm not familiar with such configuration.  
So could you share the commands to start an epmdless node and to connect the node using `remsh`?

**Comment by Zabrane** [comment]
Hi @sile 
Sure thing. Starting an `empdless` node is very easy.
First,  turn of all your Erlang nodes, and make sure `epmd` is not running:
```
$ export ERL_DIST_PORT=4369
$ epmd -kill
```
Second, add the following to your `vm.args`:
```erlang
-start_epmd false
```

When you perform the above and try to run the node, no `epmd` process is started.
```bash
$ ps auxwww | grep epmd | grep -v grep
$ 
```

To get more context, this empdless requirement was needed when our Docker nodes started misbehaving due to `epmd`. There's a [detailed discussion](https://erlangforums.com/t/epmd-becomes-zombie-when-starting-my-release-inside-docker/3036) on the Erlang forum.

**Comment by Zabrane** [comment]
@sile Amazing. Don't hesitate if you want me to test when ready.

**Comment by sile** [comment]
Just created #13 to address this issue.
You can connect to epmdless nodes by specifying the `--port $ERL_DIST_PORT` option.
Please let me know if you encounter any problems in your environment. 🙏

**Comment by Zabrane** [comment]
@sile I switched to the empdless-nodes branch and the binary rebuilt from source.

But as you can see, I'm unable to set the `--port` option:
```bash
error: unexpected argument '--port' found
  tip: to pass '--port' as a value, use '-- --port'
```

Not sure if missed something during the build.
Do you have a pre-built binary somewhere with the new `--port` option I can try?

**Comment by sile** [comment]
Maybe, the position of `--port` option seems wrong (see below):
```console
// NG
$ cargo run --port 8888 run foo@localhost
error: unexpected argument '--port' found
  tip: to pass '--port' as a value, use '-- --port'

// OK
$ cargo run run --port 8888 foo@localhost

// OK
$ cargo run run foo@localhost --port 8888
```

**Comment by Zabrane** [comment]
@sile Do you see the new `--port` option when you type:
```bash
$ erldash --help
Erlang Dashboard

Usage: erldash <COMMAND>

Commands:
  run     Run the dashboard
  replay  Replay a previously recorded dashboard session
  help    Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version

$ erldash -V
erldash 0.2.0
```

**Comment by sile** [comment]
`--port` is an option of the `run` subcommand (see below):
```console
$ erldash run -h
Run the dashboard

Usage: erldash run [OPTIONS] <ERLANG_NODE>

Arguments:
  <ERLANG_NODE>  Target Erlang node name

Options:
  -i, --polling-interval <POLLING_INTERVAL>
          Erlang metrics polling interval (in seconds) [default: 1]
  -c, --cookie <COOKIE>
          Erlang cookie
      --record <FILE>
          If specified, the collected metrics will be recorded to the given file and can be replayed later
  -p, --port <PORT>
          Port number on which the target node listens
  -h, --help
          Print help (see more with '--help')
```

**Comment by Zabrane** [comment]
@sile I see the issue.
```bash
./erldash run -h
Run the dashboard

Usage: erldash run [OPTIONS] <ERLANG_NODE>

Arguments:
  <ERLANG_NODE>  Target Erlang node name

Options:
  -i, --polling-interval <POLLING_INTERVAL>
          Erlang metrics polling interval (in seconds) [default: 1]
  -c, --cookie <COOKIE>
          Erlang cookie
      --record <FILE>
          If specified, the collected metrics will be recorded to the given file and can be replayed later
  -h, --help
          Print help (see more with '--help')
```

But following your steps above, I have the --port option now and `erldash` works like a charm :-)
Yeahh !!!

**Comment by Zabrane** [comment]
@sile When do you think you can merge the `--port` to master?

**Comment by sile** [comment]
Thank you for your confirmation!
Just merged the PR and released `erldash-0.3.0`.

**Comment by Zabrane** [comment]
@sile Amazing. Keep up the good work and long life to `erldash`
